forge-game/src/main/java/forge/game/ability/effects/DestroyAllEffect.java

```diff
@@ -76,7 +78,10 @@ public void resolve(SpellAbility sa) {
         if (sa.hasParam("RememberAllObjects")) {
             card.addRemembered(list);
         }
-
+        
+        if (isOptional && !sa.getActivatingPlayer().getController().confirmAction(sa, null, Localizer.getInstance().getMessage("lblWouldYouLikeDestroy",(card.getName())), null)) {
```

Comment:
The Card Parameter is wrong there

Reply:
Do you mean card.getName is wrong ar the message in general ?

Reply:
More in a way when you want to destroy a list of cards, then "card.getName" where card isn't even part of the list doesn't make much sense

Reply:
![image](https://user-images.githubusercontent.com/67333662/215603720-1c05b759-0590-4e1d-b1e9-9e58734068d1.png)

Reply:
I think i finally got it lol 
![image](https://user-images.githubusercontent.com/67333662/215608036-44825ffb-0fc6-4d02-9a66-cb3535ebbd23.png)